patternbuilder.install

```diff
@@ -28,14 +28,18 @@ function patternbuilder_schema() {
         'description' => 'Pattern type.',
         'type' => 'varchar',
         'length' => 64,
-        'default' => '',
         'not null' => TRUE,
       ),
+      'status' => array(
+        'description' => 'The status name of this pattern.',
+        'type' => 'varchar',
+        'length' => '255',
```

Comment:
Yes, 32 should be good. That's what is used for node type.  I'll add some validation when invoking the hook since there currently there is no restriction to what can be returned by a module.